technical-interviews/README.md

```diff
@@ -0,0 +1,72 @@
+# Technical interviews
+
+## Learning objectives
+
+```objectives
+- [ ] Perform a technical interview.
+- [ ] Explain what you're thinking when solving a problem.
+- [ ] Write code in front of someone in an interview setting.
+```
+
+You will be applying for jobs, many of which require doing a technical interview.
+
+You should already have [read about technical interviews](https://sdc.codeyourfuture.io/guides/employability/interview/technical-interview-prep/). In this workshop, you'll be practising interviewing and being interviewed about technical content.
+
+Below is a list of interview questions. You should have decided with your partner which question you are asking before coming to class, and should have prepared for it. Do not look at, or prepare for, any of the other questions.
+
+## Interview questions
+
+* [Is it a Set?](https://www.codewars.com/kata/5fddfbe6a65636001cc4fcd2)
+* [Look and say numbers](https://www.codewars.com/kata/53ea07c9247bc3fcaa00084d)
+* [Mix fruit juice](https://www.codewars.com/kata/5905871c00881d0e85000015)
+* [Multiples of 3 or 5](https://www.codewars.com/kata/514b92a657cdc65150000006)
+* [FizzBuzz Backwards](https://www.codewars.com/kata/59ad13d5589d2a1d84000020)
+
+## Preparing to interview someone
+
+Think about your goals as an interviewer. You want to:
+* See the best someone can do - you're not trying to trick them.
+* Work out how they approach problems:
+  * Do they make a plan?
+  * Do they think about edge cases?
+  * How do they handle not knowing how to do something?
+  * How easy are they to help?
+* Can they solve the problem?
+* Can they reason about their code?
+* Do they run through their code with examples to test it?
+
+To do this you will need to:
+* Already know a good solution to the problem (ideally multiple solutions!)
+* Prepare how you're going to explain the problem
+* Listen to the candidate as they're being interviewed
+  * Work out when they're stuck (but give them enough time to come up with an answer before intervening)
+  * If they're not telling you what they're thinking, encourage them to talk aloud
```

Comment:
Again, give specific prompts. Encourage them to talk aloud. Say things like: "Talk through what you are thinking" and "Can you unpack this for me a bit more."